apache-cassandra-mixin/panels.libsonnet

```
@@ -136,6 +133,7 @@ local commonlib = import 'common-lib/common/main.libsonnet';
         + g.panel.heatmap.queryOptions.withTargets([
           signals.overview.writeLatencyHeatmap.asTarget(),
         ])
+        + g.panel.heatmap.standardOptions.withUnit('s')
```

as I mentioned with a screenshot on the existing thread, units are not shown still.

For heatmaps you have to specify the axis and can't just use `standardOptions`, so it should be:

```suggestion
        + g.panel.heatmap.options.yAxis.withUnit('s')
```

[Docs here](https://github.com/grafana/grafonnet/blob/main/docs/API/panel/heatmap/index.md#fn-optionsyaxiswithunit) :)